### PR TITLE
Digital specialists multiquestions

### PR DESCRIPTION
**Pull request by allait**
Adds the specialists 'subsections' support.

Depends on:
* [x] https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/184
* [x] https://github.com/alphagov/digitalmarketplace-utils/pull/211
* [x] https://github.com/alphagov/digitalmarketplace-frameworks/pull/119

### Replace '_' with '-' in content section URLs
With the update to dmutils section_ids are now using dashes instead
of underscores.

### Remove get_service_attributes
`dmutils.service_attribute` is being replaced by `ContentManifest.summary`

### Add support for "Edit" summary column for section.edit_question
Allows displaying a link to "Add/Edit" individual questions within
the section.

### Add edit question section URLs
Specialist questions are edited separately, not as part of the whole
section. In order to support this we use the new method to transform
a multiquestion into a section object. Only questions with `slug` are
supported and the parent section must have `edit_questions` set to
`True` in order to allow individual question edits.

### CODE REVIEW

File: app/main/views/frameworks.py [hunk 1]
```diff
@@ -162,8 +162,8 @@ def framework_submission_services(framework_slug, lot_slug):
 
     for draft in itertools.chain(drafts, complete_drafts):
         draft['priceString'] = format_service_price(draft)
-        content = content_loader.get_builder(framework_slug, 'edit_submission').filter(draft)
-        sections = get_service_attributes(draft, content)
+        content = content_loader.get_manifest(framework_slug, 'edit_submission').filter(draft)
+        sections = content.summary(draft)
```
Comment (robyoung) [inline app/main/views/frameworks.py]: nice

**Comment by robyoung** [comment]
This depends on an update to the API JSON schemas.

File: app/templates/services/_base_service_page.html [hunk 2]
```diff
@@ -22,31 +22,39 @@
...
+                    <a href="{{ url_for(".edit_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id, section_id=section.id) }}">Answer required</a>
                 {% else %}
                   Not answered
                 {% endif %}
               {% endcall %}
-            {% endcall %}
-          {% else %}
-            {% call summary.row() %}
-              {{ summary.field_name(question.label) }}
+            {% elif question.is_empty%}
```
Comment (robyoung) [inline app/templates/services/_base_service_page.html]: I can't find where `question.is_empty` is defined.

Comment (robyoung) [inline app/templates/services/_base_service_page.html]: Found it!

**Comment by TheDoubleK** [comment]
I've been working off this branch and all seems to work :+1:

**Pull request merged**